openslides/utils/cache_providers.py

```diff
@@ -530,3 +565,28 @@ def get_all_cachables() -> List[Cachable]:
             continue
         out.extend(get_startup_elements())
     return out
+
+
+lua_script_change_data = """
```

Comment:
Here the same. I would prefer to indent the code for better readability.

Reply:
I don't understand what you mean with indent. See line 575 for example